.circleci/config.yml

```diff
@@ -93,6 +93,7 @@ jobs:
     docker:
       - image: zondax/rust-ci:latest
     steps:
+      - run: sudo apt-get update -y && sudo apt-get install -y libusb-1.0.0 libudev-dev
```

Comment:
Why this here? it seems this PR is bringing other Ledger related things?

Reply:
Yeah it is actually from the `ledger` branch. It has modification from this PR which has not been merged yet into `dev`.

https://github.com/Zondax/filecoin-rs/pull/103

Maybe I should change the target branch to `ledger` ?

Reply:
Yes, I know the problem is that I need to rework the Ledger branch quite a bit because to jest changes and also makes this very difficult to review.
As you know what changes you've made, I think it is easier to see if you can move over the changes on clear dev branch.